platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -250,17 +250,6 @@ private GetTransactionResponse patchTransaction(PatchTransactionRequest patch)
       case "24":
         JdbcSep24Transaction sep24Txn = (JdbcSep24Transaction) txn;
 
-        // add a memo for the transaction if the transaction is ready for user to send funds
-        if (sep24Txn.getMemo() == null
-            && Kind.WITHDRAWAL.getKind().equals(sep24Txn.getKind())
-            && sep24Txn.getStatus().equals(PENDING_USR_TRANSFER_START.toString())) {
-          SepDepositInfo sep24DepositInfo = sep24DepositInfoGenerator.generate(sep24Txn);
-          sep24Txn.setToAccount(sep24DepositInfo.getStellarAddress());
```

Comment:
`to_account` was set incorrectly before, but we should probably keep setting it as well, in case anyone was depending on it.

Reply:
Hmmm can you point me to were it was set incorrectly, or is it changed anywhere? 


This `sep24DepositInfo.getStellarAddress()` is returning the same address as "sep24Txn.toAccount()"

Reply:
`to` should be the off-chain account in case of `withdraw`, but here it's set to a Stellar address ([SEP link](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#fields-for-withdraw-transactions)).

That being said, I see that this is also set when the transaction is created, so please ignore my comment 😅